utility/fetch_modules.py

```diff
@@ -0,0 +1,57 @@
+# Copyright The IETF Trust 2022, All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = 'Dmytro Kyrychenko'
+__copyright__ = 'Copyright The IETF Trust 2022, All Rights Reserved'
+__license__ = 'Apache License, Version 2.0'
+__email__ = 'dmytro.kyrychenko@pantheon.tech'
+
+import logging
+import time
+import typing as t
+
+import requests
+
+from utility.create_config import create_config
+from utility.staticVariables import json_headers
+
+SLEEP_TIME = 30
+N_RETRIES = 5
+
+
+def fetch_modules(logger: logging.Logger) -> t.Optional[t.List[dict]]:
```

Comment:
The collections from `typing` are deprecated as of python 3.9 and will be removed in the future. `t.List[dict]` should be replaced with `list[dict]`.